app.js

```diff
@@ -1,56 +1,251 @@
+  return Math.floor(Math.random() * (itemsArray.length));
+}
+
+
+// generates 3 unique images to render on page
+// got help from Andres from class with the repl.it example in class
+function renderItems() {
+  // create array of random number that are not the same
+  // set to < 9 to make sure at least half of the images get rendered equally on page
+  while (randomNumberArray.length < 9) {
+    let item = randomItem();
+    if (!randomNumberArray.includes(item)) {
+      randomNumberArray.push(item);
+    }
   }
+  // i < randomNumberArray.length - 6 to ensure the first three images render
+  for (let i = 0; i < randomNumberArray.length - 6; i++) {
+    // assigned each of the 3 items an item number
+    let item1 = randomNumberArray.shift();
+    let image1 = document.getElementById('item1');
+    image1.src = itemsArray[item1].src;
+    image1.alt = itemsArray[item1].name;
 
-let allProducts= [
-  new Product ('bag'),  
-  new Product ('banana'),
-  new Product ('bathroom'),
-  new Product ('boots'),
-  new Product ('breakfast'),
-  new Product ('bubblegum'),
-  new Product ('chair'),
-  new Product ('cthulhu'),
-  new Product ('dog-duck'),
-  new Product ('dragon'),
-  new Product ('pen'),
-  new Product ('pet-sweep'),
-  new Product ('reset'),
-  new Product ('scissors'),
-  new Product ('shark'),
-  new Product ('sweep'),
-  new Product ('tauntaun'),
-  new Product ('unicorn'),
-  new Product ('water-can'),
-  new Product ('wine-glass'),
-];
+    let item2 = randomNumberArray.shift();
+    let image2 = document.getElementById('item2');
+    image2.src = itemsArray[item2].src;
+    image2.alt = itemsArray[item2].name;
 
-console.log(allProducts);
+    let item3 = randomNumberArray.shift();
+    let image3 = document.getElementById('item3');
+    image3.src = itemsArray[item3].src;
+    image3.alt = itemsArray[item3].name;
+
+    //counting how many times the items were viewed
+    itemsArray[item1].views++;
+    itemsArray[item2].views++;
+    itemsArray[item3].views++;
+  }
+}
 
-function randomImage() {
-  let result = Math.floor(Math.random() * allProducts.length);
-  return allProducts[result];
 
+//event handler for items and number of likes they get
+function handleLikes(event) {
+  counter++;
+  for (let i = 0; i < itemsArray.length; i++) {
+    if (event.target.alt === itemsArray[i].name) {
+      itemsArray[i].likes++;
+      break;
+    }
+  }
+  // when the counter reaches maxClicks, the results render
+  if (counter === maxClicks) {
+    images.removeEventListener('click', handleLikes);
+    renderResults();
+  } else {
+    // random items keep rendering until counter === maxClicks
+    renderItems();
+  }
 }
 
-let image = randomImage();
-console.log(image);
 
+// stores results of each voting round in localStorage
+function storeItems(arr) {
+  // KEY name is 'saveItems'
+  // stringifies objects to put into localStorage
+  let resultsData = JSON.stringify(arr);
+  localStorage.setItem('saveItems', resultsData);
```

Comment:
Try to name your data more semantically. Most things that use local storage, and most cookies (which are very similar) want to 'save items'.